packages/hsl-shared-components/src/Footer/Footer.js

```diff
@@ -1,38 +1,18 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import styled from 'styled-components/primitives';
 
-import View from '../View';
-import { Mobile, Desktop } from '../utils';
+import { Responsive } from '../utils';
 
 import FooterMobile from './FooterMobile';
 import FooterDesktop from './FooterDesktop';
 
-const Footer = styled(({ account, socialMedia, info, frontpage, links, ...rest }) => (
-  <View {...rest}>
-    <Mobile>
-      <FooterMobile
-        account={account}
-        socialMedia={socialMedia}
-        info={info}
-        frontpage={frontpage}
-        links={links}
-      />
-    </Mobile>
-    <Desktop>
-      <FooterDesktop
-        account={account}
-        socialMedia={socialMedia}
-        info={info}
-        frontpage={frontpage}
-        links={links}
-      />
-    </Desktop>
-  </View>
-))`
-  align-self: stretch;
-  align-items: stretch;
-`;
+const Footer = ({ account, socialMedia, info, frontpage, links }) => {
+  const props = { account, socialMedia, info, frontpage, links };
+  return (<Responsive
```

Comment:
The tablet view from Zeplin UIKit is missing from this setup, probably should be something like this:
`<Responsive
     small={<FooterMobile {...props} />}
     large={<FooterTablet {...props} />}
     xlarge={<FooterDesktop {...props} />}
   />`

Reply:
Also, I'm getting this error when trying to open footer via native storybook --> https://photos.app.goo.gl/bTjmIlqVedyuuoB92